### PR TITLE
Migrate contracts

### PR DESCRIPTION
**Pull request by jpc20**
# Pull request checklist

## Pull request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [ ] Feature
- [x] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## Current behavior

- We were using an old version of the contracts and the factory address was hard coded.

## New behavior

- The factory address now references the .env file and the contracts have been re-compiled and deployed to rinkeby.

## Other information

- I will send a slack message with the updated .env file

### CODE REVIEW

File: src/components/CampaignDetails.vue [hunk 1]
```diff
@@ -66,11 +66,7 @@ export default {
   },
   data() {
     return {
-<<<<<<< HEAD
```
Comment (jpc20) [inline src/components/CampaignDetails.vue:69]: leftover from a merge conflict.

Comment (attom2) [reply src/components/CampaignDetails.vue:69]: Thank you!

File: src/components/NewCampaignForm.vue [hunk 2]
```diff
@@ -53,7 +48,7 @@ export default {
         return
       }
       const accounts = await web3.eth.getAccounts();
-      const factory = await Factory.at("0x7c70286f6991c660a0cC6d52A74aEBbDE45Da380");
+      const factory = await Factory.at(process.env.VUE_APP_FACTORY_ADDRESS);
```
Comment (jpc20) [inline src/components/NewCampaignForm.vue:51]: The factory address now lives in the .env file so we only have to change it in one place when deploying a new factory.

**Review by attom2 — APPROVED** [review]
Looks good! Seems ridiculous we have to start every variable with VUE_APP

**Pull request merged**